source/using_api/index.rst

```diff
@@ -1,87 +1,59 @@
-`widgets library <http://opendatasoft.github.io/ods-widgets/docs/#/api>`_ which are only a presentation layer for the
-data provided by the APIs.
-
-These APIs allow you to search for datasets, search for data, compute analysis, perform geographic aggregation and much
-more.
-
-To access the developer API documentations, click the following links:
+.. note::
 
-- `Search API v1 <https://docs.opendatasoft.com/api/explore/v1.html>`_
-- `Search API v2 <https://docs.opendatasoft.com/api/explore/v2.html>`_
-- `OData API <https://docs.opendatasoft.com/api/explore/odata.html>`_
-- `CSW API <https://docs.opendatasoft.com/api/explore/csw.html>`_
-- `WFS API <https://docs.opendatasoft.com/api/explore/wfs.html>`_
+  To know more about what APIs are and why and how they are used, you can read the article "`You don’t need analogies to understand APIs <https://kitchen.opendatasoft.com/you-dont-need-analogies-to-understand-apis-14da4513f970>`_" written by one of our OpenDataSoft developers.
 
-Key concepts
-------------
+OpenDataSoft provides access to 6 APIs, that can be put into 3 categories:
 
-All APIs will provide you access to the same data, only in different ways. It is very important to be familiar with
-the nature of this data before diving the full specification of each API.
+- `ODS Search API v1 <https://docs.opendatasoft.com/api/explore/v1.html>`_ and `ODS Search API v2 <https://docs.opendatasoft.com/api/explore/v2.html>`_: OpenDataSoft's own APIs, used internally by the platform and the `ODS widgets <http://opendatasoft.github.io/ods-widgets/docs/#/api>`_
+- `Triple Pattern Fragments API <https://docs.opendatasoft.com/api/explore/tpf.html>`_: ODS API for triple pattern querying over datasets from OpenDataSoft portals
+- `OData <https://docs.opendatasoft.com/api/explore/odata.html>`_, `WFS <https://docs.opendatasoft.com/api/explore/wfs.html>`_ and `CSW <https://docs.opendatasoft.com/api/explore/csw.html>`_, 3 standard protocols supported and provided by OpenDataSoft
 
-Everything within the platform is related through a tree structure. Which means that you can think of OpenDataSoft as
-such:
+All these APIs provide access to any data pushed to the platform, no matter their source or format, as long as the security rules defined by the data owner allow that access. These APIs can be used, for example, to search for datasets and data, to compute analysis or to perform geographic aggregation. Still developer-oriented, but for more concrete examples of what can be done with APIs: applications as data reuses. Some of OpenDataSoft customers have indeed seen their datasets being reused through their API, to create applications (i.e. `Qui dit miam ! <http://quiditmiam.fr/>`_ and `Y'A D Frites! <https://www.opendatasoft.fr/2016/10/17/y-a-d-frites-application-avec-de-lopen-data-dedans/>`_ 2 applications that display the school canteen menu of French cities, Toulouse, Rennes and Saint-Malo).
```

Comment:
`to perform geographic aggregations`? 

Maybe split this block into 2 or 3 paragraphs (like security, usage, users)?